src/Components/Counter.jsx

```diff
@@ -1,15 +1,43 @@
-import React from 'react';
-import Display from './Display';
+import React, {useState} from 'react';
+import  "./DisplayCss.css";
+import Display from "../Components/Display"
 import ButtonPad from './ButtonPad';
 
 
-function Counter() {
+const Counter=()=> {
+    const [count, setCount] = useState(0);
+    const [list, setList]= useState([]);
+    const handleCount = (handle)=>{
+        if(handle == 0){
+            setCount(0)
+            setList([0])
+            return 
+        }
+        setCount(count+handle);
+        let newItem = count+handle;
+        let newList = list;
```

Comment:
do these need to be let's, or can they be const?  always make the default variable types to be const, then become flexible as the needs arise.